### PR TITLE
chore(eks): kubectl 1.16.8-eks-e16311, aws-cli 1.18.86, helm 3.2.4

### LINKED ISSUE #7216: [EKS] Upgrade to kubectl 1.15 & AWS CLI 1.18.70
bump `helm3` and  `kubectl` versions from the latest release of **aws-samples/aws-lambda-layer-kubectl** [2.0.0-beta3](https://github.com/aws-samples/aws-lambda-layer-kubectl/releases/tag/2.0.0-beta3)

### Use Case

1. we need the support to create new namespace with `addChart()` in aws-eks.
https://github.com/aws/aws-cdk/issues/3748#issuecomment-610053250

2. Amazon EKS 1.15 has been released for a while. Let's bump the kubectl version to 1.15.

### Other

* [x] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by pahud**
- bump aws-lambda-layer-kubectl [2.0.0](https://github.com/aws-samples/aws-lambda-layer-kubectl/releases/tag/2.0.0)
with kubectl v1.16.8-eks-e16311, awscli 1.18.86 and helm 3.2.4
- update the helm repo of the kubernetes-dashboard in the integ test to `https://kubernetes.github.io/dashboard/` from [Helm Hub](https://hub.helm.sh/charts/k8s-dashboard) as `https://kubernetes-charts.storage.googleapis.com` has been deprecated

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by pahud** [comment]
Hi @eladb , I am ready.

File: packages/@aws-cdk/aws-eks/lib/helm-chart.ts [hunk 1]
```diff
@@ -53,6 +53,12 @@ export interface HelmChartOptions {
    * @default Duration.minutes(5)
    */
   readonly timeout?: Duration;
+
+  /**
+   * create namespace if not exist
+   * @default False
+   */
+  readonly createNamespace?: boolean;
```
Comment (eladb) [inline packages/@aws-cdk/aws-eks/lib/helm-chart.ts:61]: Sorry but this should be a separate PR :-) 

[think about the changelog entry :-)]

_I added comments on the namespace change as well here so it's PR can start from there_

File: packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py [hunk 2]
```diff
@@ -20,22 +21,23 @@ def helm_handler(event, context):
...
-    role_arn     = props['RoleArn']
-    release      = props['Release']
-    chart        = props['Chart']
-    version      = props.get('Version', None)
-    wait         = props.get('Wait', False)
-    timeout      = props.get('Timeout', None)
-    namespace    = props.get('Namespace', None)
-    repository   = props.get('Repository', None)
-    values_text  = props.get('Values', None)
+    role_arn = props['RoleArn']
```
Comment (eladb) [inline packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py:24]: Please avoid aesthetic changes, those make the code review harder.

Any reason spacing was changed?

File: packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py [hunk 3]
```diff
@@ -20,22 +21,23 @@ def helm_handler(event, context):
...
+    values_text = props.get('Values', None)
 
     # "log in" to the cluster
-    subprocess.check_call([ 'aws', 'eks', 'update-kubeconfig',
-        '--role-arn', role_arn,
-        '--name', cluster_name,
-        '--kubeconfig', kubeconfig
-    ])
+    subprocess.check_call(['aws', 'eks', 'update-kubeconfig',
+                           '--role-arn', role_arn,
```
Comment (eladb) [inline packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py:37]: Ditto

File: packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts [hunk 4]
```diff
@@ -14,8 +14,11 @@ class EksClusterStack extends TestStack {
       assumedBy: new iam.AccountRootPrincipal(),
     });
 
+    const vpc = new ec2.Vpc(this, 'Vpc',  { maxAzs: 3, natGateways: 1});
```
Comment (eladb) [inline packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts:17]: * add a comment explaining why this is needed

* style:

```suggestion
    const vpc = new ec2.Vpc(this, 'Vpc',  { maxAzs: 3, natGateways: 1 });
```

File: packages/@aws-cdk/aws-eks/lib/helm-chart.ts (hunk 1 above)
Comment (eladb) [inline packages/@aws-cdk/aws-eks/lib/helm-chart.ts:61]: Feels like it makes more sense for this to be enabled by default, no? What are the implications?

**Comment by pahud** [comment]
I'm still working on the integ tests. Got lambda timeout error.

description in https://github.com/aws/aws-cdk/pull/8773#issuecomment-650915370

**Comment by eladb** [comment]
Merged just for you!

**Comment by pahud** [comment]
Hi @eladb this PR should be ready now.
